forge-gui/res/cardsfolder/upcoming/cathartic_parting.txt

```diff
@@ -0,0 +1,6 @@
+Name:Cathartic Parting
+ManaCost:1 G
+Types:Sorcery
+A:SP$ ChangeZone | Origin$ Battlefield | Destination$ Library | ValidTgts$ Artifact.OppCtrl,Enchantment.OppCtrl | TgtPrompt$ Select target artifact or enchantment an opponent controls | SubAbility$ DBChangeZone | Shuffle$ True | SpellDescription$ The owner of target artifact or enchantment an opponent controls shuffles it into their library. You may shuffle up to four target cards from your graveyard into your library.
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Library | ValidTgts$ Card.YouOwn | TgtPrompt$ Select target card from your graveyard | TargetMin$ 0 | TargetMax$ 4 | Shuffle$ True

```

Comment:
Seems like the "may" still needs to be addressed:
But should it allow skipping of only some of the targets?
Or do you think rather just offer to not shuffle at all?

Reply:
I've noticed that problem, Release notes just say
"You may target zero cards in your graveyard and still choose to shuffle your library."

The card I used as template (Devious Cover-Up) had the same problem.

Reply:
I'll fix it in a follow-up